Sources/StreamChatUI/CommonViews/PlayPauseButton/PlayPauseButton.swift

```diff
@@ -0,0 +1,50 @@
+//
+// Copyright © 2023 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+open class PlayPauseButton: MediaButton {
```

Comment:
I can see that both `PillButton` and `PlayPauseButton` are subclasses of `MediaButton`. This goes against the considered best practices for the UI SDK, as mentioned [here](https://www.notion.so/stream-wiki/Writing-a-new-UI-Component-UIKit-5aeb6a53d7404079b5051f3ee7f17e41?pvs=4#32e4b15769ef40b3b36bb2a4d23c32ff). 

The main issue with this approach is that if a customer wants to customize the `MediaButton` only and for example change the highlighted logic, this change won't be reflected in all the buttons that are subclassing the `MediaButton`. So if a customer does `Components.default.mediaButton = CustomMediaButton.self` this won't change all buttons who subclass it.

I can see that for this particular scenario, it is tricky since we are handling a Button 🤔 For composition, a Button is tricky because we can't have a Button inside another Button 🤔 Or can we? Maybe we can forward the events of the Parent button to the underlying button?